MetaMask/metamask-extension
# Fix permission list shrinking with less text

## Explanation
Fixes an issue where the entire permissions list would shrink if the permission text was small.

## More information

@holantonela requested this change while reviewing #13613 but opening a seperate PR to keep the scope of the other PR smaller.

## Screenshots/Screencaps



### Before



![image](https://user-images.githubusercontent.com/1561200/158626278-b32ce2ca-d500-4966-bac3-9bfc7db5e88f.png)

### After



![image](https://user-images.githubusercontent.com/1561200/158626194-409cdcff-d95c-4e36-87ff-606b1ffad7b2.png)
![image](https://user-images.githubusercontent.com/1561200/158626162-8db69f0b-52b5-434a-806e-7e9cd6c077d4.png)


## Manual testing steps

- Start the DApp connect process with any DApp
- Try to manually reduce the amount of text in the permissions list
- See that it doesn't shrink anymore!
- The same can be done for the Snap Install process (use https://filsnap.chainsafe.io/)
